docs/blog/2024-09-12-next-gen-module-federation-deployments.md

```diff
@@ -0,0 +1,549 @@
+    asset 377.090696af.js 63.4 KiB [emitted] [immutable] (id hint: vendors)
+  asset main.4833bb53.js 77.9 KiB [emitted] [immutable] (name: main)
+  asset 553.ffd2bced.js 26.1 KiB [emitted] [immutable]
+  asset 964.db21dd7d.js 6.33 KiB [emitted] [immutable]
+assets by path *.json 7.75 KiB
+  asset mf-stats.json 4.04 KiB [emitted]
+  asset mf-manifest.json 3.71 KiB [emitted]
+asset favicon.ico 14.7 KiB [emitted]
+asset 3rdpartylicenses.txt 4.38 KiB [emitted]
+asset index.html 311 bytes [emitted]
+asset assets/.gitkeep 0 bytes [emitted]
+runtime modules 12 KiB 19 modules
+orphan modules 307 KiB [orphan] 18 modules
+cacheable modules 614 KiB (javascript) 132 bytes (share-init) 126 bytes (consume-shared) 6 bytes (remote)
+  javascript modules 614 KiB 21 modules
+  provide-module modules 126 bytes
+    provide shared module (default) react-dom@18.3.1 = /Users/columferry/dev/nrwl/issues/zephyr-blog/myorg/node_modules/react-dom/index.js 42 bytes [built] [code generated]
+    provide shared module (default) react-router-dom@6.11.2 = /Users/columferry/dev/nrwl/issues/zephyr-blog/myorg/node_modules/react-router-dom/dist/index.js 42 bytes [built] [code generated]
+    provide shared module (default) react@18.3.1 = /Users/columferry/dev/nrwl/issues/zephyr-blog/myorg/node_modules/react/index.js 42 bytes [built] [code generated]
+  consume-shared-module modules 126 bytes
+    consume shared module (default) react@18.3.1 (strict) (strict) (fallback: /Users/columferry/dev/nrwl/issues/zephyr-blog/myorg/node_modules/react/index.js) 42 bytes [built] [code generated]
+    consume shared module (default) react-router-dom@6.11.2 (strict) (strict) (fallback: /Users/columferry/dev/nrwl/issues/zephyr-blog/myorg/node_modules/react-router-dom/dist/index.js) 42 bytes [built] [code generated]
+    consume shared module (default) react-dom@18.3.1 (strict) (strict) (fallback: /Users/columferry/dev/nrwl/issues/zephyr-blog/myorg/node_modules/react-dom/index.js) 42 bytes [built] [code generated]
+  remote remote1/Module 6 bytes (share-init) 6 bytes (remote) [built] [code generated]
+
+WARNING in ⚠ DefinePlugin:
+  │ Conflicting values for 'process.env.NODE_ENV' ("'production'" !== "\"production\"")
+
+
+Rspack 1.0.4 compiled with 1 warning in 1.29 s
+```
+
+{% callout type="note" title="Zephyr Cloud Account Creation" %}
+At this point, a page will likely be opened in your browser where you can create your Zephyr Cloud account and
+authenticate the CLI to allow it to deploy the applications to your account.
+{% /callout %}
+
+You'll note that the output from this is not the same as the output we received earlier when we ran the builds.
+In particular, you should look out for the following logs (_Note: your URL should be different_):
+
+```shell
+ ZEPHYR   https://colum_nrwl_io_6-remote1-nx-zephyr-cloud-example-c-e7ab3d770-ze.zephyrcloud.app
+ ZEPHYR   https://colum_nrwl_io_7-shell-nx-zephyr-cloud-example-col-04ccb4027-ze.zephyrcloud.app
+```
```

Comment:
I think this needs to be a little clearer. Like "not the same as the output we received earlier": when earlier?

Also the URLs you post here. One doesn't immediately connect that they are an excerpt from the above log output.